corehq/apps/translations/integrations/transifex/views.py

```diff
@@ -233,9 +233,17 @@ def page_context(self):
             context['pull_resource_form'] = self.pull_resource_form
         return context
 
-    @staticmethod
-    def _generate_excel_file(transifex, resource_slug, target_lang):
-        wb = Workbook(write_only=True)
+    def _generate_excel_file(self, domain, resource_slug, target_lang):
+        """
+        extract translations from po file pulled from transifex and converts to a xlsx file
+
+        :return: Workbook object
+        """
+        transifex = Transifex(domain=domain, app_id=None,
```

Comment:
where is this used?

Reply:
Line 249

Reply:
aha, i missed that you changed the argument.

Reply:
Good call.